realm/src/androidTest/java/io/realm/RealmInMemoryTest.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
I havn't checked, but how much copy/paste is there?
I would assume that testing the inMemory Realm or the file based can be done generically using just a parameter more for relevant tests?